scripts/setupTypeScriptRollup.js

```diff
@@ -0,0 +1,137 @@
+// @ts-check
+const fs = require("fs")
+const path = require("path")
+const { argv } = require("process")
+
+const projectRoot = argv[2] || path.join(__dirname, "..")
+
+console.log("Adding typescript with rollup")
+
+// Add deps to pkg.json
+let pkgJSONPath = path.join(projectRoot, "package.json")
+const packageJSON = JSON.parse(fs.readFileSync(pkgJSONPath, "utf8"))
+packageJSON.devDependencies = Object.assign(packageJSON.devDependencies, {
+	"svelte-check": "^1.0.0",
```

Comment:
it might be nice to keep these alphabetized with the `@` deps coming first

Reply:
Haha, I actually had it alphabetized but switched it back to conform with the `svelte/template` one.